conda_build/build.py

```diff
@@ -1026,6 +1026,16 @@ def tests_failed(m, move_broken, broken_dir, config):
 
 def check_external():
     if sys.platform.startswith('linux'):
+        ldd = external.find_executable('ldd')
+        if ldd is None:
+            sys.exit("""\
+Error:
+    Did not find 'ldd' in: %s
+    'ldd' is necessary for building conda packages on Linux with
+    relocatable ELF libraries.  Please install with your system package
```

Comment:
Also if you were curious about musl, they provide their own `ldd` as well. Alpine Linux will expose it in the same standard way too.

xref: http://wiki.musl-libc.org/wiki/FAQ#Q:_where_is_ldd_.3F
xref: https://pkgs.alpinelinux.org/contents?branch=edge&name=musl-utils&arch=x86_64&repo=main